src/table/tableview/table-view.component.js

```diff
@@ -92,6 +83,17 @@ angular.module('patternfly.table').component('pfTableView', {
           ctrl.pageConfig.pageNumber = 1;
         }
       }
+    }
+
+    ctrl.$onInit = function () {
+
+      if (ctrl.debug) {
+        $log.debug("$onInit");
+      }
```

Comment:
Yes, I've been wanting to, but @akieling always seems to have an open table PR ( #572 ) when I'm also doing a PR, so I'm trying to avoid merge issues  -which would happen if I remove all the debug stuff.